src/ng/parse.js

```diff
@@ -1763,6 +1768,24 @@ function $ParseProvider() {
     return this;
   };
 
+    /**
+   * @ngdoc method
+   * @name $parseProvider#$$exposeAst
+   * @description
+   *
+   * Configures the access to the generated AST for parsed expressions.
+   * This feature is not meant to be part of the public API, so use it at your own risk.
+   *
+   * @param {boolean} exposeAst if set to true, the AST of each parsed expression will be exposed through
+   * an `$$ast` property added to the $parse's return value.
+   *
+   **/
+  this.$$exposeAst = function(exposeAst) {
+    if (isBoolean(exposeAst)) {
+      globalExposeAst = exposeAst;
+    }
+  };
```

Comment:
It might be better to stick to the getter/setter model we use in other places:

```js
this.$$exposeAst = function(exposeAst) {
  if (isUndefined(exposeAst)) {
    // Getter
    return globalExposeAst;
  } else {
    // Setter
    globalExposeAst = !!exposeAst;   // or: (exposeAst === true);
    return this;
  }
}
```